scripts/release.sh

```diff
@@ -8,4 +8,3 @@ fi
 
 git tag $1
 git push origin $1
-gh release create $1 --draft --generate-notes --title "$1" release/*.tgz
```

Comment:
Good catch! Today, the release process looks like the below:

- Add a commit that updates plugin.yaml to have a new version
- Run this release.sh

After this and #517, the process would look like:

- Add a commit that updates plugin.yaml to have a new version
- Create a semver tag along with the release on the GitHub web UI
- The workflow gets triggered on GHA. goreleaser does the rest.

I guess we'd rather prefer the below:

- Add a commit that updates plugin.yaml to have a new version
- A GHA workflow reads the new plugin.yaml version value and tags the commit with the version
- The release workflow gets triggered and goreleaser does the rest.

Maybe we can modify this release.sh to do the second bullet point?
Basically, we can deduce `$1` here from `plugin.yml` by reading the version field using eg. `yq`? 🤔